websocket_server/Config.js

```diff
@@ -31,7 +32,11 @@ const Config = {
 
 	BYPASS_SSL_VALIDATION: Utils.parseBooleanFromEnv(process.env.BYPASS_SSL_VALIDATION),
 
-	PORT: process.env.PORT || DEFAULT_PORT,
+	IS_DEV: Utils.parseBooleanFromEnv(process.env.IS_DEV),
+
+	SETUP_TYPE: process.env.SETUP_TYPE || DEFAULT_SETUP_TYPE,
```

Comment:
Not a blocker: I'm wondering if we could detect this from any of the env variables that app_api injects so we don't need to set this explicitly on the docker image. I would prefer if we could have a single multi purpose docker image.

Reply:
I did a similar thing with the APP_PORT below, as the previously hardcoded one 23000 did not work if you had another ex app installed before.